### PR TITLE
add Ji's avatar

### PR DESCRIPTION
**Pull request by LiuJi-Jim**

### CODE REVIEW

File: components/TeamSection.vue [hunk 1]
```diff
@@ -123,6 +123,11 @@ const people = [
     role: "Engineering Intern",
     imageUrl: "zilong.webp",
   },
+  {
+    name: "Ji",
+    role: "Senior Frontend Engineer",
```
Comment (tianzhou) [inline components/TeamSection.vue:128]: Let's just say frontend engineer?

Comment (tianzhou) [reply components/TeamSection.vue:128]: Or maybe just Engineer

Comment (LiuJi-Jim) [reply components/TeamSection.vue:128]: > Or maybe just Engineer

Yes, "Engineer" is good.

**Review by tianzhou — APPROVED** [review]
LGTM. Welcome to the party!

**Pull request merged**